src/sidebar/panel.js

```diff
@@ -6,16 +6,29 @@ const formats = [
   'strike',
   'header',
   'indent',
-  'list'
+  'list',
+  'direction',
+  'align'
```

Comment:
Align must be allowed for direction to work, as shown below.

Without align: 
<img width="394" alt="image" src="https://user-images.githubusercontent.com/9160471/27442946-aa3599ca-573f-11e7-95b8-0e3e19e2c7da.png">


With align
<img width="366" alt="image" src="https://user-images.githubusercontent.com/9160471/27442966-ba9f7a6a-573f-11e7-9257-84d174d229af.png">